Operators/Types/user/Dom/mapping/SquareImage_9dc87ab7-85cd-4f88-9d52-f323f8df87c3.t3ui

```diff
@@ -0,0 +1,110 @@
+{
+  "Id": "9dc87ab7-85cd-4f88-9d52-f323f8df87c3"/*SquareImage*/,
+  "Description": "turn a given input image into a square of its largest resolution",
+  "InputUis": [
```

Comment:
This is a great idea for a new Library op. But I would make it more generic like [ChangeTextureSize] with a bunch of modes like: Fit, Cover, etc. like the new Layer2d.